From 9b240fa7f2d23ff312881153682ea367075c9625 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Tue, 27 Nov 2012 18:27:23 -0600
Subject: ENGR00235081 Quad DL: Fix chip select for SPI-NOR and flags

Fix chip select for SPI-NOR and
remove flags for no writeable partition for weim nor and
SPI-NOR

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index e191e1d..156c553 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -409,7 +409,6 @@ static struct mtd_partition m25p32_partitions[] = {
 		.name	= "bootloader",
 		.offset	= 0,
 		.size	= SZ_256K,
-		.mask_flags = MTD_WRITEABLE,
 	}, {
 		.name	= "bootenv",
 		.offset = MTDPART_OFS_APPEND,
@@ -435,7 +434,7 @@ static struct spi_board_info m25p32_spi0_board_info[] __initdata = {
 		.modalias	= "m25p80",
 		.max_speed_hz	= 20000000,
 		.bus_num	= 0,
-		.chip_select	= 1,
+		.chip_select	= 0,
 		.platform_data	= &m25p32_spi_flash_data,
 	},
 };
@@ -450,7 +449,6 @@ static struct mtd_partition mxc_nor_partitions[] = {
 		.name	= "bootloader",
 		.offset	= 0,
 		.size	= SZ_256K,
-		.mask_flags = MTD_WRITEABLE,
 	}, {
 		.name = "bootenv",
 		.offset = MTDPART_OFS_APPEND,
-- 
1.7.9.5

